interfaces/webgl2.idl

```diff
@@ -177,8 +177,6 @@ interface mixin WebGL2RenderingContextBase
   const GLenum FRAMEBUFFER_ATTACHMENT_DEPTH_SIZE             = 0x8216;
   const GLenum FRAMEBUFFER_ATTACHMENT_STENCIL_SIZE           = 0x8217;
   const GLenum FRAMEBUFFER_DEFAULT                           = 0x8218;
-  const GLenum DEPTH_STENCIL_ATTACHMENT                      = 0x821A;
```

Comment:
The `DEPTH_STENCIL_ATTACHMENT` and `DEPTH_STENCIL` constants were removed here but not added anywhere else. There are 4 fewer subtests as a result. Seems like maybe a mishap when editing the spec, need to check this before merging.

Reply:
This is still the case.

Reply:
Found the spec change, this was intentional: https://github.com/KhronosGroup/WebGL/pull/2873